src/test/java/SeleniumFramework.java

```diff
@@ -1,51 +1,72 @@
-import org.junit.Test;
+import org.testng.annotations.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
 
-import java.sql.Time;
-import java.util.concurrent.TimeUnit;
 
 public class SeleniumFramework {
 
     @Test
-    public void passwordCheck() {
+    public static void LoginSuccess() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
-        driver.get("https://deens-master.now.sh/login");
+        driver.get("https://deens.com/?internaltraffic");
```

Comment:
Please use maximaze method for driver window or handle this element 
![image](https://user-images.githubusercontent.com/51605259/68787689-0a1dd180-0653-11ea-9441-3fea411ece34.png)